ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -85,6 +86,12 @@ all:
                     mgmt_interface_fabric:
                     mgmt_interface_host:
                     mgmt_interface_platform:
+            TESTS_PLATFORM:
```

Comment:
do we need all this to test a new platform? Normally we just assign the platform to one of the existing devices in any scenario

Reply:
it is needed to check for MLAG counters as I need a group of 2 hosts to generate the mlag config - I tried to include it in the L2Leaf under TESTS but could not succeed - happy to change if there is a way. The advantage of cretaing this other node group is that it makes it possible to get some group_vars and so to generate smaller configs.

Of course happy to change it if there is a better way

Reply:
Following our discussions just pushed a new commit leveraging DC1-CL1A to test for the new platform variables hence reducing the new amount of generated configruation